History.md

```diff
@@ -3,6 +3,7 @@
 ### Unreleased
 
 * Split Strainer class as a factory and a template (#1208) [Thierry Joyal]
+* Remove safe navigation (#1218) [Thierry Joyal]
```

Comment:
It won't be clear what this is referring to without actually looking at this pull request.  Perhaps "Remove handling of a nil context in the Strainer class".

Reply:
Updated code and PR description